api/api/spec/spec.yaml

```diff
@@ -5687,7 +5687,8 @@ paths:
                     type: string
                     format: alphanumeric
                   force_time:
-                    description: "Remove the old agent with the same IP if disconnected since <force_time> seconds"
+                    description: "Remove the old agent with the same name or IP if disconnected for <force_time>
```

Comment:
```suggestion
                    description: "Remove the old agent with the same name, ID or IP if disconnected for <force_time>
```

Reply:
In this case (`POST /agents`), the only accepted parameters are `name` and `ip`. The ID can't be chosen and Wazuh uses the next available one. Therefore, I think force_time wouldn't have an impact on said param.